### ISSUE TITLE
Prompt for locale on synthetics init

### ISSUE THREAD

**Issue by andrewvc** [elastic/synthetics#786]
As mentioned [in this discuss thread](https://discuss.elastic.co/t/feedback-regarding-synthetics-tests/335587/6?u=andrew_cholakian1) it's confusing to not have a proper locale set when running synthetics checks. Really all users should set this to some explicit value. The best way we can do this is to have the `init` command prompt on setup. We should probably set the timezone as well. If we do that it would compliment https://github.com/elastic/synthetics/issues/777 well.

**Comment by wolframhaussig** [comment]
I agree that setting the locale should be done at init time and the current terminal locale is a sensible default.

I am just not sure if I want the same to be true for the Timezone:
When running monitors in two different locations - e.g. US and Germany - I wouldn't expect to have the German time for the United States data. would it be possible to have the timezone set on the location and to use the terminal timezone for local tests only?
